PUBLISH.md

```diff
@@ -0,0 +1,42 @@
+# Publish guide
```

Comment:
We should also call out updating the changelog. Additionally, we should also consider introducing the quick process of documenting released on github for client libraries through https://github.com/plaid/plaid-node/releases/new

Reply:
⚡️